assets/src/skate.d.ts

```diff
@@ -15,6 +15,8 @@ export interface Route {
 
 export type RouteId = string
 
+export type ScheduleAdherenceStatus = "on-time" | "early" | "late"
```

Comment:
When would you use a string constant like this vs an enum?

Reply:
I wish enums worked better, but apparently in TS exporting an enum doesn't work properly. (At least in some cases? If you try to export an enum in this file the TS compile fails.) It seems like the advice is just to use strings like this.